src/main/java/org/opensearch/knn/index/query/ExactSearcher.java

```diff
@@ -71,15 +106,17 @@ private Map<Integer, Float> scoreAllDocs(KNNIterator iterator) throws IOExceptio
         return docToScore;
     }
 
-    private Map<Integer, Float> searchTopK(KNNIterator iterator, int k) throws IOException {
+    private Map<Integer, Float> searchTopCandidates(KNNIterator iterator, int limit, @NonNull Predicate<Float> filterScore)
```

Comment:
Curious why k variable is renamed to limit?

Reply:
This method will be called by radialSearch as well. Hence, renamed it to K to be more generic.